docs/sphinx/targets/python/pasqal.py

```diff
@@ -0,0 +1,65 @@
+import cudaq
+from cudaq.operator import *
+
+# This example illustrates how to use Pasqal's EMU_MPS emulator over Pasqal's cloud via CUDA-Q.
+
+# To obtain the authentication token we recommend logging in
+# by using Pasqal's Python SDK.
+# Fill in the  username and password via the environment variables.
+# The password can be left empty in an interactive session to be
+# prompted to enter the it securely via the command line interface.
+#
+# Contact Pasqal at help@pasqal.com or through https://community.pasqal.com for assistance.
+#
+# See our general docs https://docs.pasqal.com/cloud/set-up/
+# to see how to get this setup, or the CUDA-Q documentation at
+# https://nvidia.github.io/cuda-quantum/latest/using/backends/hardware/neutralatom.html#pasqal
+from pasqal_cloud import SDK
+import os
+
+sdk = SDK(
+    username=os.environ.get("PASQAL_USERNAME"),
+    password=os.environ.get("PASQAL_PASSWORD", None),
```

Comment:
Are we saving a plain password in an environment variable? I'm not sure this is secure. If the Pasqal sdk already deals with passwords in a secure way then maybe we can have the sdk deal directly with passwords.

Reply:
I'd recommend not using the environment variable (hence it defaults to None) and getting it.
However, I don't think that would work in a Jupyter Notebook for example hence I'd propose to keep it.